lib/utils.js

```diff
@@ -270,13 +270,17 @@ module.exports.deleteDir = function (dir_path) {
   }
 };
 
+/**
+ * Returns absolute filename to save screenshot.
+ * @param fileName {string} - filename.
+ */
 module.exports.screenshotOutputFolder = function (fileName) {
   const fileSep = path.sep;
 
   if (!fileName.includes(fileSep) || fileName.includes('record_')) {
```

Comment:
@DavertMik , I don't understand why this condition is used.
Both `!fileName.includes(fileSep)` and `record_` look like something strange.

Is need to change `return path.join(global.output_dir, fileName);` to `path.resolve` too?

Reply:
the `record_` was added in this PR -> https://github.com/Codeception/CodeceptJS/commit/5b8b2bef285b34815b3a05427e56a9de859f51c3#diff-50e3aa130a4f97a42ee2cf111c7b1d9dR274

Reply:
> So far so good. But fileName.includes(fileSep) should be replaced with something more meaningful.

Yep, looks like it should be more meaningful to avoid such questions